2022-01-08 18:04:55,611 CRIT Supervisor running as root (no user in config file)
2022-01-08 18:04:55,613 INFO supervisord started with pid 1
2022-01-08 18:04:56,615 INFO spawned: 'nginx' with pid 15
2022-01-08 18:04:56,616 INFO spawned: 'php7-fpm' with pid 16
2022-01-08 18:04:56,656 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139795212367024 for <Subprocess at 139795212368144 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-08 18:04:56,657 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139795212367024 for <Subprocess at 139795212368144 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-08 18:04:57,658 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-08 18:04:57,659 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-08 18:50:39,861 WARN received SIGTERM indicating exit request
2022-01-08 18:50:39,861 INFO waiting for nginx, php7-fpm to die
2022-01-08 18:50:39,870 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139795212367024 for <Subprocess at 139795212368144 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-08 18:50:39,881 INFO stopped: php7-fpm (exit status 0)
2022-01-08 18:50:39,885 INFO stopped: nginx (exit status 0)
2022-01-11 21:07:18,552 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:07:18,556 INFO supervisord started with pid 1
2022-01-11 21:07:19,558 INFO spawned: 'nginx' with pid 10
2022-01-11 21:07:19,560 INFO spawned: 'php7-fpm' with pid 11
2022-01-11 21:07:19,721 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140365092140208 for <Subprocess at 140365092141328 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:07:19,723 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140365092140208 for <Subprocess at 140365092141328 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:07:19,723 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140365092140208 for <Subprocess at 140365092141328 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:07:20,724 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:07:20,725 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:08:30,988 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:08:30,990 INFO supervisord started with pid 1
2022-01-11 21:08:31,992 INFO spawned: 'nginx' with pid 15
2022-01-11 21:08:31,993 INFO spawned: 'php7-fpm' with pid 16
2022-01-11 21:08:32,043 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140032536655024 for <Subprocess at 140032536656144 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:08:32,058 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140032536655024 for <Subprocess at 140032536656144 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:08:32,058 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140032536655024 for <Subprocess at 140032536656144 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:08:33,059 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:08:33,060 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:27:46,476 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:27:46,477 INFO supervisord started with pid 1
2022-01-11 21:27:47,479 INFO spawned: 'nginx' with pid 16
2022-01-11 21:27:47,480 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 21:27:47,531 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140422254826672 for <Subprocess at 140422254827792 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:27:47,538 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140422254826672 for <Subprocess at 140422254827792 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:27:48,539 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:27:48,540 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:32:56,522 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:32:56,551 INFO supervisord started with pid 1
2022-01-11 21:32:57,553 INFO spawned: 'nginx' with pid 16
2022-01-11 21:32:57,555 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 21:32:57,601 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140718899291312 for <Subprocess at 140718899292432 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:32:57,610 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140718899291312 for <Subprocess at 140718899292432 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:32:58,611 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:32:58,611 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:48:24,225 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:48:24,226 INFO supervisord started with pid 1
2022-01-11 21:48:25,228 INFO spawned: 'nginx' with pid 16
2022-01-11 21:48:25,229 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 21:48:25,276 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140275627353264 for <Subprocess at 140275627354384 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:48:25,310 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140275627353264 for <Subprocess at 140275627354384 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:48:26,312 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:48:26,312 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:59:07,244 WARN received SIGTERM indicating exit request
2022-01-11 21:59:07,244 INFO waiting for nginx, php7-fpm to die
2022-01-11 21:59:07,245 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140275627353264 for <Subprocess at 140275627354384 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:07,248 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140275627353264 for <Subprocess at 140275627354384 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:07,265 INFO stopped: php7-fpm (exit status 0)
2022-01-11 21:59:07,270 INFO stopped: nginx (exit status 0)
2022-01-11 21:59:14,825 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:59:14,827 INFO supervisord started with pid 1
2022-01-11 21:59:15,829 INFO spawned: 'nginx' with pid 15
2022-01-11 21:59:15,830 INFO spawned: 'php7-fpm' with pid 16
2022-01-11 21:59:15,876 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140333112837296 for <Subprocess at 140333112838416 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:15,877 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140333112837296 for <Subprocess at 140333112838416 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:16,878 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:59:16,878 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:59:38,053 WARN received SIGTERM indicating exit request
2022-01-11 21:59:38,053 INFO waiting for nginx, php7-fpm to die
2022-01-11 21:59:38,053 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140333112837296 for <Subprocess at 140333112838416 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:38,054 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140333112837296 for <Subprocess at 140333112838416 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:38,061 INFO stopped: php7-fpm (exit status 0)
2022-01-11 21:59:38,063 INFO stopped: nginx (exit status 0)
2022-01-11 21:59:52,485 CRIT Supervisor running as root (no user in config file)
2022-01-11 21:59:52,487 INFO supervisord started with pid 1
2022-01-11 21:59:53,488 INFO spawned: 'nginx' with pid 16
2022-01-11 21:59:53,490 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 21:59:53,544 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140415113929904 for <Subprocess at 140415113931024 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:53,545 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140415113929904 for <Subprocess at 140415113931024 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:53,545 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140415113929904 for <Subprocess at 140415113931024 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 21:59:54,546 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 21:59:54,546 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:03:56,821 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:03:56,824 INFO supervisord started with pid 1
2022-01-11 22:03:57,865 INFO spawned: 'nginx' with pid 21
2022-01-11 22:03:57,867 INFO spawned: 'php7-fpm' with pid 23
2022-01-11 22:03:57,919 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139821613679792 for <Subprocess at 139821613680912 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:03:57,920 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139821613679792 for <Subprocess at 139821613680912 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:03:58,921 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:03:58,921 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:04:59,129 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:04:59,149 INFO supervisord started with pid 1
2022-01-11 22:05:00,151 INFO spawned: 'nginx' with pid 17
2022-01-11 22:05:00,152 INFO spawned: 'php7-fpm' with pid 18
2022-01-11 22:05:00,197 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139761332434096 for <Subprocess at 139761332435216 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:05:00,202 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139761332434096 for <Subprocess at 139761332435216 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:05:01,203 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:05:01,203 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:14:00,067 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:14:00,068 INFO supervisord started with pid 1
2022-01-11 22:14:01,070 INFO spawned: 'nginx' with pid 16
2022-01-11 22:14:01,071 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 22:14:01,118 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140129985275056 for <Subprocess at 140129985276176 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:14:01,132 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140129985275056 for <Subprocess at 140129985276176 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:14:02,133 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:14:02,133 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:17:34,603 WARN received SIGTERM indicating exit request
2022-01-11 22:17:34,603 INFO waiting for nginx, php7-fpm to die
2022-01-11 22:17:34,603 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140129985275056 for <Subprocess at 140129985276176 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:17:34,604 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140129985275056 for <Subprocess at 140129985276176 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:17:34,609 INFO stopped: php7-fpm (exit status 0)
2022-01-11 22:17:34,611 INFO stopped: nginx (exit status 0)
2022-01-11 22:18:17,659 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:18:17,660 INFO supervisord started with pid 1
2022-01-11 22:18:18,662 INFO spawned: 'nginx' with pid 16
2022-01-11 22:18:18,663 INFO spawned: 'php7-fpm' with pid 17
2022-01-11 22:18:18,710 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139920223765680 for <Subprocess at 139920223766800 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:18:18,711 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139920223765680 for <Subprocess at 139920223766800 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:18:19,712 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:18:19,712 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:25:14,713 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:25:14,715 INFO supervisord started with pid 1
2022-01-11 22:25:15,717 INFO spawned: 'nginx' with pid 15
2022-01-11 22:25:15,718 INFO spawned: 'php7-fpm' with pid 16
2022-01-11 22:25:15,763 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140377136090240 for <Subprocess at 140377135258752 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:25:15,764 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140377136090240 for <Subprocess at 140377135258752 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:25:16,765 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:25:16,765 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:27:02,980 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:27:02,982 INFO supervisord started with pid 1
2022-01-11 22:27:04,009 INFO spawned: 'nginx' with pid 17
2022-01-11 22:27:04,010 INFO spawned: 'php7-fpm' with pid 22
2022-01-11 22:27:04,053 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140146570745008 for <Subprocess at 140146570746128 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:04,054 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140146570745008 for <Subprocess at 140146570746128 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:05,055 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:27:05,055 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:27:28,715 WARN received SIGTERM indicating exit request
2022-01-11 22:27:28,715 INFO waiting for nginx, php7-fpm to die
2022-01-11 22:27:28,715 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140146570745008 for <Subprocess at 140146570746128 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:28,716 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140146570745008 for <Subprocess at 140146570746128 with name php7-fpm in state STOPPING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:28,722 INFO stopped: php7-fpm (exit status 0)
2022-01-11 22:27:28,729 INFO stopped: nginx (exit status 0)
2022-01-11 22:27:40,434 CRIT Supervisor running as root (no user in config file)
2022-01-11 22:27:40,435 INFO supervisord started with pid 1
2022-01-11 22:27:41,438 INFO spawned: 'nginx' with pid 15
2022-01-11 22:27:41,439 INFO spawned: 'php7-fpm' with pid 16
2022-01-11 22:27:41,486 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140442690745520 for <Subprocess at 140442690746640 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:41,487 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140442690745520 for <Subprocess at 140442690746640 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-01-11 22:27:42,487 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-01-11 22:27:42,487 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:30:56,706 CRIT Supervisor running as root (no user in config file)
2022-02-07 10:30:56,709 INFO supervisord started with pid 1
2022-02-07 10:30:57,711 INFO spawned: 'nginx' with pid 10
2022-02-07 10:30:57,713 INFO spawned: 'php7-fpm' with pid 11
2022-02-07 10:30:57,743 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140304049591472 for <Subprocess at 140304049592592 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:30:57,745 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140304049591472 for <Subprocess at 140304049592592 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:30:58,746 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:30:58,747 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:32:27,873 CRIT Supervisor running as root (no user in config file)
2022-02-07 10:32:27,876 INFO supervisord started with pid 1
2022-02-07 10:32:28,878 INFO spawned: 'nginx' with pid 10
2022-02-07 10:32:28,879 INFO spawned: 'php7-fpm' with pid 11
2022-02-07 10:32:28,907 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139962623243440 for <Subprocess at 139962623244560 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:32:28,908 CRIT uncaptured python exception, closing channel <POutputDispatcher at 139962623243440 for <Subprocess at 139962623244560 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:32:29,909 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:32:29,910 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:33:05,378 CRIT Supervisor running as root (no user in config file)
2022-02-07 10:33:05,381 INFO supervisord started with pid 1
2022-02-07 10:33:06,384 INFO spawned: 'nginx' with pid 10
2022-02-07 10:33:06,386 INFO spawned: 'php7-fpm' with pid 11
2022-02-07 10:33:06,411 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140539066135728 for <Subprocess at 140539066136848 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:33:06,412 CRIT uncaptured python exception, closing channel <POutputDispatcher at 140539066135728 for <Subprocess at 140539066136848 with name php7-fpm in state STARTING> (stderr)> (<type 'exceptions.IOError'>:[Errno 29] Illegal seek [/usr/lib/python2.7/dist-packages/supervisor/supervisord.py|runforever|227] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|handle_read_event|232] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|record_output|166] [/usr/lib/python2.7/dist-packages/supervisor/dispatchers.py|_log|142] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|info|275] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|log|293] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|emit|186] [/usr/lib/python2.7/dist-packages/supervisor/loggers.py|doRollover|211])
2022-02-07 10:33:07,414 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-02-07 10:33:07,414 INFO success: php7-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
